source/_data/glossary.yml

```
@@ -60,6 +60,8 @@
   description: "[Scripts](/docs/scripts/) are components that allow users to specify a sequence of actions to be executed by Home Assistant when turned on."
 - topic: Sensor
   description: "[Sensors](/integrations/sensor) return information about a thing, for instance the level of water in a tank."
+- topic: Selectors
+  description: "[Selectors](/docs/blueprint/selectors/) specify what values are accepted for a blueprint input and how the input is shown in the user interface."
```

```suggestion
  description: "[Selectors](/docs/blueprint/selectors/) specify what values are accepted for a Blueprint input and how the input is shown in the user interface."
```

Not just Blueprints, they also apply to scripts, automations, and services.
They don't specify values, but they provide UI features/input.

---

Changed. I took one of the sentence from the official documentation.
https://www.home-assistant.io/docs/blueprint/selectors/
See #23737